frontend/public/module/k8s/types.ts

```diff
@@ -470,6 +470,11 @@ export type StorageClassResourceKind = {
   parameters?: {
     [key: string]: string;
   };
+  resource?: {
+    parameters?: {
+      encrypted: string;
+    };
+  };
```

Comment:
this is wrong, plz check correct struct for encrypted StorageClass...

Reply:
did u test this for encrypted StorageClasses, how was it working for you when this struct is not correct ??

Reply:
does ODF StorageClasses have different struct from non-ODF ones ??

Reply:
define correct type of "param" here: https://github.com/openshift/console/blob/master/frontend/public/components/utils/storage-class-dropdown.tsx#L262 and import that in restore/clone etc files along with the dropdown.